packages/service/src/db.test.ts

```diff
@@ -1,26 +1,51 @@
-import * as Y from 'yjs';
+import * as Y from "yjs";
+import db from "./utils/db";
 
-test("crdts can be persisted",()=>{
-	// create a yjs document A
-	const ydoc = new Y.Doc();
-	// make a change to the document
-	const ytext = ydoc.getText("codemirror");
-	ytext.insert(0,"hello world");
-	const state = Y.encodeStateAsUpdate(ydoc)
+afterAll(() => {
+  db.$disconnect();
+});
 
-	// persist this change to the database
-	// ????
+beforeAll(async () => {
+  await db.document.deleteMany({
+    where: {
+      origin: { contains: "test doc" },
+    },
+  });
+});
 
-	// get the change back from the database
-	const stateFromDatabase = state
-	
-	// create yjs document B from database change
-	const ydoc2 = new Y.Doc();
-	Y.applyUpdate(ydoc2, stateFromDatabase);
+test("crdts can be persisted", async () => {
+  // create a yjs document A
+  const ydoc = new Y.Doc();
+  // make a change to the document
+  const ytext = ydoc.getText("codemirror");
+  ytext.insert(0, "hello world");
+  const state = Y.encodeStateAsUpdate(ydoc);
 
-	// expect that the contents of document A matches document B
-	// test the result
-	expect(ydoc.getText('codemirror').toJSON()).toBe(
-		ydoc2.getText('codemirror').toJSON()
-	)
-});
\ No newline at end of file
+  // persist this change to the database
+  await db.document.create({
+    data: {
+      value: Buffer.from(state),
+      origin: "test doc",
```

Comment:
let's flip these around, the value of the document should be the value of the computed CRDT. The encoded CRDT itself should live in the `origin` field.

Reply:
When you say the value of the computed CRDT corresponds to the value field, are you referring to the unencoded CRDT (so just plain document text)?